typings/sharify.d.ts

```diff
@@ -20,7 +20,6 @@ declare module "sharify" {
       readonly APP_URL: string
       readonly ARTIST_COLLECTIONS_RAIL?: string // TODO: remove after CollectionsRail a/b test
       readonly ARTIST_COLLECTIONS_RAIL_IDS: string[]
-      readonly CLIENT_NAVIGATION: string // TODO: remove after AB test
```

Comment:
Any reason to remove all the test-supporting code? I imagine we'll want to relaunch it with the primary fix.

Reply:
This is the name of the AB test, which will likely need to be renamed when we relaunch.

The other lines are removing the 'Experiment Viewed' event, which still fires even though the AB test is disabled. This is unnecessarily noisy plus eats up into our available Segment quota.